### PR TITLE
Fix appstream

### PR DESCRIPTION
**Pull request by darkdragon-001**

### CODE REVIEW

**Comment by darkdragon-001** [comment]
@barthalion Is it not possible to combine `git` with `patch` source? It reports a schema error...

**Comment by barthalion** [comment]
If there's anything wrong with the schema that seems to be allowed as per flatpak-manifest man page, the best is to fix it directly in the flatpak-builder repo.

File: appstream-developer.patch [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+@@ -21,6 +21,13 @@
+     <launchable type="desktop-id">org.gnome.Calls.desktop</launchable>
+     <translation type="gettext">calls</translation>
+ 
++  <!-- developer_name tag deprecated with Appstream 1.0 -->
++  <developer_name>Julian Sparber, Evangelos Ribeiro Tzaras</developer_name>
++    <developer id="puri.sm">
++      <name translatable="no">Julian Sparber</name>
++      <name translatable="no">Evangelos Ribeiro Tzaras</name>
++    </developer>
```
Comment (bbhtt) [inline appstream-developer.patch:25]: @darkdragon-001 Only one name child tag is allowed in source locale form https://github.com/ximion/appstream/issues/607

Comment (darkdragon-001) [reply appstream-developer.patch:25]: This is a backport of https://gitlab.gnome.org/GNOME/calls/-/merge_requests/711. Please report upstream.

**Pull request merged**